### PR TITLE
#AV-102732: Java SDK release script changes

### PR DESCRIPTION
**Pull request by chaitanyaavi**

### CODE REVIEW

File: create_release.sh [hunk 1]
```diff
@@ -64,9 +79,23 @@ else
     exit 1
 fi
 
+# Release Avi's Java SDK jar file
+
+if [ -z $AVISDK_PGP_PASSPHRASE ]; then
+  echo "Cannot release java SDK environment is not set"
```
Comment (grastogi23) [inline create_release.sh]: we should say that AVISDK_PGP_PASSPHRASE is not set. So anyone running it knows without looking at the code.

**Review by grastogi23 — CHANGES_REQUESTED** [review]

File: create_release.sh [hunk 2]
```diff
@@ -79,8 +108,12 @@ git push -f origin $REL
 cd ..
 rm -rf avitools
 
+
 /usr/local/bin/hub release $hub_op $assets -F ReleaseNote $REL_TAG
```
Comment (grastogi23) [inline create_release.sh:112]: while we are at refactoring can we also check for output of the hub command and test for success. Only if the hub command is successful do rest of the following cleanups

Comment (chaitanyaavi) [reply create_release.sh:112]: Cleanups are harmless even if the release is not done removing will not error out

**Review by grastogi23 — APPROVED** [review]

File: create_release.sh (hunk 1 above)
Comment (grastogi23) [reply create_release.sh]: we should also exit from here. not just be in else.

File: create_release.sh [hunk 3]
```diff
@@ -64,9 +79,23 @@ else
     exit 1
 fi
 
+# Release Avi's Java SDK jar file
+
+if [ -z $AVISDK_PGP_PASSPHRASE ]; then
+  echo "Cannot release java SDK AVISDK_PGP_PASSPHRASE is not set"
```
Comment (grastogi23) [inline create_release.sh:85]: There should be an exit here. There is no reason to continue the release if the java cannot be released.

**Review by grastogi23 — CHANGES_REQUESTED** [review]

**Review by manojkumarjain — APPROVED** [review]
Approve

**Pull request merged**